Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG BASEIMAGE=rust:latest
+
+# BUILDER PATTERN
+
+FROM $BASEIMAGE AS builder
+
+WORKDIR /builder
+
+COPY . .
+
+RUN apt-get update
+RUN apt-get -y install libclang-dev libmetis-dev libscotch-dev
+
+ARG C_INCLUDE_PATH="/usr/include/x86_64-linux-musl"
+ARG CPLUS_INCLUDE_PATH="/usr/include/x86_64-linux-musl"
+
+RUN rustup target add x86_64-unknown-linux-musl
+RUN cargo build --workspace --release --bins --target x86_64-unknown-linux-musl --no-default-features
+
+# FINAL IMAGE
+
+FROM alpine
+
+WORKDIR /coupe
+
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/apply-part /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/apply-weight /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/medit2svg /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/mesh-part /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/mesh-refine /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/mesh-reorder /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/part-bench /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/part-info /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/weight-gen /bin
+
```

Comment:
Can be hard to maintain if we have to always list all targets. The good side is that if we remove or rename a target this container build will fail.

Reply:
Should we add a build argument to specify a target ?

Note : that would lead to keeping `ARG C_INCLUDE_PATH=/usr/include/${TARGET}` and `ARG CPLUS_INCLUDE_PATH=/usr/include/${TARGET}` right ?

Reply:
I went for something else and the target is no more.
If you think it could be a good thing to have some `VAR TARGET` we should probably add it to the `workflow strategy matrix config` right ?